core/src/consensus/solo/mod.rs

```diff
@@ -60,7 +60,7 @@ where
     }
 
     fn generate_seal(&self, _block: &M::LiveBlock, _parent: &M::Header) -> Seal {
```

Comment:
What does `Some` means?

Reply:
Before this commit, the seal type is Regular OR Proposal OR None. Now there only remains Regular. So I changed the seal type to Option.